code/_helpers/atmospherics.dm

```diff
@@ -20,6 +20,7 @@
 	var/list/results = list()
 	if (total_moles>0)
 		results += "<span class='notice'>Pressure: [round(pressure,0.1)] kPa</span>"
+		results += "<span class='notice'>Moles: [round(total_moles,0.1)]</span>"
```

Comment:
New convention for span class is to do SPAN_X (SPAN_NOTICE in this instance) instead of <span class = 'blah'></span>. So it would be 
```suggestion
		to_chat(user, SPAN_NOTICE("Results of the analysis[src == A ? "" : " of [A]"]"))
```
You don't HAVE to swap these <span>s out when messing with older code but you definitely should not be adding more of them.

Reply:
Though it's fine this time because the compiler gets really angry about ternaries used in SPAN_X procs.
If this ever happens to you, just set a var before the line.
for example: var/results = "Results of the analysis[src == A ? "" : " of [A]"]"
to_chat(user, SPAN_NOTICE(results))